tool/src/org/antlr/v4/codegen/target/DartTarget.java

```diff
@@ -36,7 +36,9 @@ public class DartTarget extends Target {
 		"deferred", "hide", "return", "with",
 		"do", "if", "set", "yield",
 
-		"rule", "parserRule"
+                "rule", "parserRule",
+                "accept"
```

Comment:
Please fix indentation and use tabs instead of spaces.

Reply:
I will make the change, but we really should not use tabs. Tab expansion display is dependent on the editor used. In GNU Emacs, it's 8 spaces. For Github, it's 4 spaces. For VSCode, 4. For vim, 8. It is well known: [tabs are evil](https://www.emacswiki.org/emacs/TabsAreEvil). There are many places where spaces are used for indentation already. Is there a coding style doc for Antlr?

Reply:
> I will make the change, but we really should not use tabs.

Sorry, but historically ANTLR code base was written with tabs and currently it's not rational to reformat all files. But much worse is to mix spaces and tabs withing the same file.

>  Is there a coding style doc for Antlr?

For java and stg files indentation style is specified: [indent_style = tab](https://github.com/antlr/antlr4/blob/dev/.editorconfig#L10).

>  Tab expansion display is dependent on the editor used. In GNU Emacs, it's 8 spaces. For Github, it's 4 spaces. For VSCode, 4. For vim, 8.

I suppose all tools should support [editorconfig](https://github.com/antlr/antlr4/blob/dev/.editorconfig#L4). Width of tab is specified there and it's 4 spaces.